spec/miq_ae_service_spec.rb

```diff
@@ -38,29 +38,28 @@
 
 describe MiqAeMethodService::MiqAeService do
   context "#service_model" do
-    let(:workspace) { double('ws', :persist_state_hash => {}) }
+    let(:root_hash) { {'ae_state_max_retries' => '100', 'ae_retry_interval' => 2.seconds } }
```

Comment:
@lfu it would be nice to call this variable root_obj so it would match with the get_obj_from_path => root_obj